tomcat/src/main/java/org/apache/coyote/http11/Http11Processor.java

```diff
@@ -29,19 +40,89 @@ public void process(final Socket connection) {
         try (final var inputStream = connection.getInputStream();
              final var outputStream = connection.getOutputStream()) {
 
-            final var responseBody = "Hello world!";
-
-            final var response = String.join("\r\n",
-                    "HTTP/1.1 200 OK ",
-                    "Content-Type: text/html;charset=utf-8 ",
-                    "Content-Length: " + responseBody.getBytes().length + " ",
-                    "",
-                    responseBody);
+            final var requestData = getRequestData(inputStream);
+            log.info("requestData : {}", requestData);
+            tomcatController.handleRequest(requestData);
+            final var response = getResponse(requestData);
```

Comment:
handleRequest메서드에서 요청을 처리하고 응답을 반환할 것 같았어요.

tomcatController.handleRequest(requestData); 으로 요청을 처리했는데
final var response = getResponse(requestData); 에서 동일한 요청을 처리하는 게 어색해보여요.

컨트롤러에 데이터를 전달하는 RequestData.java 클래스가 존재하는데,
응답하는 ResponseData.java 클래스가 없는 게 어색합니다.

```suggestion
            final ResponseData responseData = tomcatController.handleRequest(requestData);
            
            final String response = buildResponse(responseData);
            // or 
            final String response = responseData.build();
```

Reply:
인정합니다.
급하게 개발하느라 좀 흐름이 이상했네요ㅠㅠ

tomcatController라는 클래스를 RequestHandler로 변경하고 HttpResponse를 반환하도록 변경했습니다.
requestHandler는 request를 보고 적절한 method로 연결해준뒤, 요청받은 정적 파일을 읽어오거나, 로그인 로그를 찍는 것 같은 비지니스로직을 실행하는 클래스로 만들었습니다.
HttpResponse는 필요한 값들을 저장하고, 응답을 작성하거나 byte로 변경하는 책임을 가지도록 했습니다.

이렇게 하니 제가 지정하지 않은 url은 접근할 수 없어서 더 보안이 좋아지고, 책임도 잘 나뉘어진 것 같습니다. 야호